app/renderer/components/Inspector/Actions.js

```diff
@@ -78,19 +74,19 @@ export default class Actions extends Component {
       <Row gutter={16} className={InspectorStyles['arg-row']}>
         <Col span={24}>
           <Select onChange={(actionGroupName) => selectActionGroup(actionGroupName)} placeholder={t('Select Action Group')}>
-            { _.keys(actionDefinitions).map((actionGroup) => <Option key={actionGroup}>{t(actionGroup)}</Option>) }
+            { _.keys(actionDefinitions).map((actionGroup) => <Select.Option key={actionGroup}>{t(actionGroup)}</Select.Option>) }
           </Select>
         </Col>
       </Row>
       {selectedActionGroup && <Row>
         <Col span={24}>
           <Select onChange={(actionGroupName) => selectSubActionGroup(actionGroupName)} placeholder={t('Select Sub Group')}>
-            { _.keys(actionDefinitions[selectedActionGroup]).map((actionGroup) => <Option key={actionGroup}>{t(actionGroup)}</Option>) }
+            { _.keys(actionDefinitions[selectedActionGroup]).map((actionGroup) => <Select.Option key={actionGroup}>{t(actionGroup)}</Select.Option>) }
```

Comment:
Some lines here are really long

Reply:
Yeah, would like to do lint stuff as another pr